src/applications/mhv-landing-page/tests/medications/e2e/medications-landing-page-feature-toggle-false.cypress.spec.js

```diff
@@ -0,0 +1,48 @@
+import vamcEhr from '../../fixtures/vamc-ehr.json';
+
+const medicationFeatureToggles = (enabled = true) => ({
+  data: {
+    type: 'feature_toggles',
+    features: [
+      {
+        name: 'mhv_medications_to_va_gov_release',
+        value: enabled,
+      },
+    ],
+  },
+});
+
+describe('Medications Landing Page', () => {
+  beforeEach(() => {
+    cy.intercept('GET', '/data/cms/vamc-ehr.json', vamcEhr).as('vamcEhr');
+    const redirectPath = '/health-care/refill-track-prescriptions';
+    cy.intercept('GET', redirectPath, '').as('refillRedirect');
+  });
+
+  it('visits Medications landing Page', () => {
+    // Login when feature disabled, expect redirect
+    cy.intercept(
+      'GET',
+      '/v0/feature_toggles*',
+      medicationFeatureToggles(false),
+    );
+    cy.login();
+    cy.visit('my-health/about-medications/');
+
+    cy.wait('@refillRedirect');
+    cy.url().should('include', '/health-care/refill-track-prescriptions');
```

Comment:
Because our build won't include this page, it's enough to wait for the redirect. Attempting to check the url will quite possibly lead to some flakiness. I think it's safe to remove this line.